snooty/diagnostics.py

```diff
@@ -883,3 +883,15 @@ def __init__(
         super().__init__(
             f"""Facet specified is not a valid facet: {facet_name}""", start, end
         )
+
+
+class NestedProject(Diagnostic):
+    severity = Diagnostic.Level.warning
+
+    def __init__(
+        self,
+        nested_project: str,
+        start: Union[int, Tuple[int, int]],
+        end: Union[None, int, Tuple[int, int]] = None,
+    ):
+        super().__init__(f"""Nested project detected: {nested_project}""", start, end)
```

Comment:
Do we want more info here, as in letting the user know that the parser will not parse the nested project?